bear-requirements.yaml

```diff
@@ -1,3 +1,4 @@
+overrides: coala-build.yaml
```

Comment:
Can you add a comment to the top of this?  Just note that it is a generated file and should not be edited by hand.

I am a bit worried because of the explicit use of `CommentedMap` for the three blocks within the file?

If we cant (easily) have comments at the top level, that is ok.

Reply:
We can comments anywhere in the file, the yaml reader just ignores the comments completely.

Reply:
We can also add comments programmatically as described here: http://yaml.readthedocs.io/en/latest/example.html